src/main/java/six/six/gateway/twilio/TwilioSmsService.java

```diff
@@ -0,0 +1,19 @@
+package six.six.gateway.twilio;
+
+import com.twilio.Twilio;
+import com.twilio.rest.api.v2010.account.Message;
+import com.twilio.type.PhoneNumber;
+import org.jboss.logging.Logger;
+import six.six.gateway.SMSService;
+
+public class TwilioSmsService implements SMSService {
+  private static final Logger logger = Logger.getLogger(TwilioSmsService.class);
+
+  @Override
+  public boolean send(String phoneNumber, String message, String login, String pw) {
+    Twilio.init(login, pw);
+
+    Message sms = Message.creator(new PhoneNumber(phoneNumber), new PhoneNumber("+15815015124"), message).create();
+    return true;
```

Comment:
What is expected here? Should we watch errors and return 'false' if sending the message failed for some reason?

Reply:
How far do we want to go? In the typical example code they only record the id of the SMS, which can be used later on to monitor the status of the sending. Behind the scenes, this seems to be an asynchronous API that accepts callbacks and creates a resource that can be monitored. Doing so seems overkill to me, since the phone is already verified and the user can request a new code if he doesn't receive anything.
Statuses that are returned are:
`    QUEUED("queued"),
    SENDING("sending"),
    SENT("sent"),
    FAILED("failed"),
    DELIVERED("delivered"),
    UNDELIVERED("undelivered"),
    RECEIVING("receiving"),
    RECEIVED("received"),
    ACCEPTED("accepted"),
    SCHEDULED("scheduled"),
    READ("read"),
    PARTIALLY_DELIVERED("partially_delivered");`

No idea what we receive, but I can imagine they send a lot of 'ACCEPTED' or 'QUEUED'. In which cases should I return false? It could change 10 ms after I return the boolean.

I can check whether there is an error already present and if there is, log it and return false. That might be a good middle way between doing nothing and overkill.

Please don't look at the rest of the code of this open source plugin ;-)